etc/configs/fake.json

```diff
@@ -10,7 +10,10 @@
             "model": "fake",
             "attributes": {
                 "arm-model": "xArm6"
-            }
+            },
```

Comment:
I assume these changes to `fake.json` are from your local debugging; mind removing them?

Reply:
yup! will remove